packages/esm-appointments-app/src/appointments/forms/create-edit-form/appointments-form.component.tsx

```diff
@@ -392,7 +392,7 @@ const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment, patientU
           ))}
       </Select>
 
-      <div className={styles.inputContainer} id="radio-group">
+      {/*      <div className={styles.inputContainer} id="radio-group">
```

Comment:
This functionality currently does nothing (the value collected is not mapped to any backing field).

An argument (perhaps a strong one) could be made for just removing this all-together, but I stuck with just commenting out  If there's no plan at all to implement this in the near future, we should just remove this.

Reply:
Right, that would a be *great* feature to add, to so let's just comment out the question until we get a chance to implement the functionality behind it, so users don't think patients will be getting emails when they won't be?